i3_agenda/config.py

```diff
@@ -89,7 +92,7 @@
     "--limchar",
     "-l",
     type=int,
-    default=-1,
+    default=MIN_CHAR_NB,
```

Comment:
what is `NB`?

Reply:
NB is for NUMBER.

Reply:
can be renamed to `MIN_CHARS` or `MIN_CHARS_COUNT` whichever you like more

Reply:
done in 59787be2